### PR TITLE
2nd Review 

### PR DESCRIPTION
**Pull request by Jayasimha099**
Please do the 2nd review as soon as possible. We are ready to integrate the application

### CODE REVIEW

**Comment by Jayasimha099** [comment]
I have added required changes. Next we are going to integrate the application.

File: project1/application.py [hunk 1]
```diff
@@ -111,8 +115,68 @@ def logout():
...
+
+	if "username" in session:
+			username = session["username"]
+			user_reviews = reviewRate.query.filter_by(Isbn = book_isbn).all()
+			res = requests.get("https://www.goodreads.com/book/review_counts.json",
+                           params={"key": "fKvLN1nI7uY5XcyXi7VgvQ", "isbns": book_isbn})
+			if request.method == "GET":
+				revie = reviewRate.query.filter(reviewRate.Isbn.like(book_isbn),reviewRate.Username.like(username)).first()
+
+				#user_reviews = reviewRate.query.filter_by(isbn=bookisbn).all()
```
Comment (ashishk144) [inline project1/application.py:160]: remove commented code.

File: project1/application.py [hunk 2]
```diff
@@ -111,8 +115,68 @@ def logout():
...
+		if request.method == "GET":
+			return render_template("search.html")
+		else:
+			result = request.form.get("search")
+			result = '%'+result+'%'
+			search_result = Books.query.filter(or_(Books.tittle.ilike(result), Books.author.ilike(result), Books.isbn.ilike(result),Books.year.ilike(result))).all()
+			if len(search_result) == 0:
+				return render_template("search.html",message = "No records found")
+			return render_template("search.html", books=search_result)
+			#return "success"
```
Comment (ashishk144) [inline project1/application.py:142]: remove commented code.

File: project1/application.py [hunk 3]
```diff
@@ -111,8 +115,68 @@ def logout():
...
+				reviews = request.form.get("Review")
+
+				isbn = book_isbn
+				username = username
+				
+				user = reviewRate(Isbn=book_isbn,Review = reviews, Rating=rating,Username = username)
+				db.session.add(user)
+				db.session.commit()
+
+			#user_reviews = reviewRate.query.filter_by(isbn = bookisbn).all()
```
Comment (ashishk144) [inline project1/application.py:179]: remove commented code

**Review by ashishk144 — COMMENTED** [review]
Why is search related code in review branch

File: project1/application.py (hunk 1 above)
Comment (Jayasimha099) [reply project1/application.py:160]: I have removed search method and removed commented code also

File: project1/application.py (hunk 2 above)
Comment (Jayasimha099) [reply project1/application.py:142]: removed commented code

File: project1/application.py (hunk 3 above)
Comment (Jayasimha099) [reply project1/application.py:179]: removed commented code

**Pull request closed without merging**